lstchain/io/io.py

```diff
@@ -355,21 +389,23 @@ def write_mcheader(mcheader, output_filename, obs_id=None, filters=None, metadat
     """
 
     extramc = ExtraMCInfo()
-    extramc.prefix = ''  # get rid of the prefix
+    extramc.prefix = ""  # get rid of the prefix
     if metadata is not None:
         add_global_metadata(mcheader, metadata)
         add_global_metadata(extramc, metadata)
 
-    with HDF5TableWriter(filename=output_filename, group_name="simulation", mode="a", filters=filters) as writer:
+    with HDF5TableWriter(
+        filename=output_filename, group_name="simulation", mode="a", filters=filters
+    ) as writer:
         extramc.obs_id = obs_id
         writer.write("run_config", [extramc, mcheader])
 
 
-@deprecated('09/07/2020', message='this function will disappear in lstchain v0.7')
+@deprecated("09/07/2020", message="this function will disappear in lstchain v0.7")
 def write_array_info_08(subarray, output_filename):
     """
     Write the array info to a ctapipe v0.8 compatible DL1 HDF5 file
-    This is a temporary solution until we move to ctapipe v0.9.1. 
+    This is a temporary solution until we move to ctapipe v0.9.1.
```

Comment:
We are already using `ctapipe>=0.10.3`

Reply:
True! Should I just change the commented message or also the name of the function?

Reply:
I think this should just be removed and replaced by `subarray.to_hdf`

Reply:
This could (even should) be done in another PR